scripts/beck/st/error_injection.py

```diff
@@ -1,66 +1,78 @@
-#!/usr/bin/env python3
-#
-# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
Why is the copy right herder removed?

Reply:
Added again. Mistakenly copied local file